packages/graphback-core/src/plugin/GraphbackPluginEngine.ts

```diff
@@ -21,26 +21,20 @@ export class GraphbackPluginEngine {
     private plugins: GraphbackPlugin[];
     private metadata: GraphbackCoreMetadata;
 
-    public constructor(schema: GraphQLSchema | string, config: GraphbackGlobalConfig) {
+    public constructor(schemaComposer: SchemaComposer<any>, config: GraphbackGlobalConfig) {
```

Comment:
More typings: NO <any>

Reply:
Should I revert the function signature to be `public constructor(schema: GraphQLSchema | string, config: GraphbackGlobalConfig)`?